CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Removed
 
 ### Bug Fixes
+- WaitForDeployment and WaitForOperatorDeployment from pkg/test/e2eutil successfully complete waiting when the available replica count is greater than the minimum replica count required. ([#2248](https://github.com/operator-framework/operator-sdk/pull/2248))
```

Comment:
```suggestion
- Fix `WaitForDeployment` and `WaitForOperatorDeployment` from`pkg/test/e2eutil` successfully complete waiting when the available replica count are equals or greater than the minimum replica count required. ([#2248](https://github.com/operator-framework/operator-sdk/pull/2248))
```

Reply:
I'm not sure this is a bug fix. Seems more like a change/improvement to me.

Also to build on @camilamacedo86's suggestion, WDYT of this for the CHANGELOG line:
```suggestion
- Updated `pkg/test/e2eutil.WaitForDeployment()` and `pkg/test/e2eutil.WaitForOperatorDeployment()` to successfully complete waiting when the available replica count is _at least_ (rather then exactly) the minimum replica count required. ([#2248](https://github.com/operator-framework/operator-sdk/pull/2248))
```

Reply:
That sounds good to me. I've updated the changelog. Thank you both for the help!